### ISSUE TITLE
Improve README with information on how to have multiple Swift versions

### ISSUE THREAD

**Issue by dmyers** [neonichu/chswift#9]
How do you have Swift 1.1 and 1.2 installed? I upgraded Xcode and when I run `chswift` all I have is 1.2.

Would you just download the non Mac App Store version from the [Apple Developer](https://developer.apple.com/downloads/index.action) site?

Would it be good to add instructions to the readme?

**Comment by neonichu** [comment]
The best way to do this is to use https://github.com/neonichu/xcode-install to install multiple versions of Xcode. Using the Mac App Store will always overwrite your current Xcode.

``` bash
$ [sudo] gem install xcode-install
$ xcode-install install 6.2
$ xcode-install install 6.3.2
$ xcode-install install '7 beta'
```

should give you 1.1, 1.2 and 2.0 via chswift.

I'm short on time right now, so would really appreciate a PR which adds instructions to the README.

**Comment by dmyers** [comment]
Thanks @neonichu. I had no idea that existed. I really like that solution the best. I ended up manually downloading another Xcode version and got it working with `chswift`, but I will be using that from now on.

I'm not sure if this project's readme needs to include how to use `xcode-install` as much as a general notice on how to manage them and a link off to that project. It's up to you, but I made a small PR if you think it's a good starting point.

Thank you for your time and explaining.
